content/_guides/creatingchannels.md

```diff
@@ -1,52 +1,196 @@
 ---
 title: Creating Channels
 category: chanop
-credits: web7
 ---
 
+A person who joins an _empty_ and unregistered channel is granted
+**temporary** Operator status. This does not guarantee their ownership of the
+channel. They need to `REGISTER` the channel with `ChanServ` to gain ownership
+of it.
+
+This guide will help you prepare for and go through that process.
+
 ## Choosing the correct namespace
 
 Before creating a new channel on Libera.Chat, please take a moment to read up
 on [our namespaces policy](/chanreg) and ensure that your channel is named
 appropriately from the beginning. This will avoid disruptions in the future.
 
-## How to create a new channel
+## Be identified
 
-### Be identified
+To register and manage channels, you will need to be [identified to](registration#logging-in)
+`NickServ`.
 
-You must be registered and logged in to services to make channels. You can log
-in by doing `/msg NickServ IDENTIFY YourNickname YourPassword`
+## Pick a name
 
-### Joining the channel
+Before doing that, you'll obviously need to pick a channel name! Think of one
+that is descriptive of the channel's purpose, and has no spaces or commas.
+The name you choose will become what's called your namespace. You can use
+some punctuation such as `.`, `_` or `+` instead of spaces, but it is better
+to just drop the space. It is typical to reserve `-` as a delimiter for
+additional channels within the Namespace, such as `-dev` or `-ops`.
 
-The next step of creating a channel is to `/join` it.
+Once you're settled on the name, add the appropriate number of `#`'s to the
+beginning, according to the rules in [our namespaces policy](/chanreg).
+
+For example, if it is an official representation of your project and the
+project is named "Awesome App", you can use one `#`. You'll probably go with
+something like: `#AwesomeApp`
+
+If it's not a "Project" or "Community" channel based on the definitions
+described in [our namespaces policy](/chanreg) you'd go with two `#`. As an
+example, it might be a channel for your friends to talk to you in, so you'd
+probably pick something like `##YourNickname`
+
+## Check that it is available
+
+You can check that a channel is registered or not with `ChanServ`. To see if
+for example `#libera` is registered, you would do:
+
+```irc
+/msg ChanServ info #libera
+```
+
+Since it is registered, ChanServ will tell you the registration details you're
+allowed to see. Otherwise it would say:
+
+```irc
+ChanServ: #libera is not registered.
+```
 
-Pick a channel name without spaces and add the appropriate number of `#`'s to
-the beginning, according to [our namespaces policy](/chanreg).
-Eg, `/join #awesomeprojectname` if it is for an official project channel
-or a community channel about a project, or `/join ##yournickname-chats` for an
-informal channel.
+## Joining the channel
 
-### Check that it is available
+The next step of creating a channel is to `/join` it.
+
+```irc
+/join <#channelname>
```

Comment:
I liked that you used an example instead of (or in addition to) a placeholder like here. I'm wondering if it would be good to have a running example channel or always use either placeholders or the example channel or both.

Reply:
I decided to change that to avoid establishing a camp-able channel for people to be taunted in and confused by if they just copy-paste verbatim. Which they will...